int32 cameraName
float32 approxRange
bool firstTarget

